common/import_.py

```diff
@@ -132,6 +132,15 @@ def link_technosphere_by_activity_hash_ref_product(
     )
 
 
+def search_activity(base_db, activity):
```

Comment:
```suggestion
def search_activity(activity: str, default_db: str | None = None):
```
As the `activity` is always used and required, I would prefer this signature to be used, and if default_db is not provided and activity doesn’t contain `:` you could throw an exception

Reply:
good point, done ✅